f3dex2.s

```diff
@@ -499,6 +502,44 @@ wait_dpc_start_valid:
     mtc0    $2, DPC_END
 f3dzex_0000111C:
     sw      $2, lbl_00F0
+.else // UCODE_METHOD == METHOD_XBUS
+wait_dpc_start_valid:
+    mfc0 $11, DPC_STATUS
+    andi $11, $11, DPC_STATUS_DMA_BUSY | DPC_STATUS_START_VALID 
+    bne $11, $zero, wait_dpc_start_valid
+     sw $zero, lbl_00F0
+    addi $11, $zero, 2
```

Comment:
This is `DPC_STATUS_SET_XBUS`, the following could be added to `rsp/rsp_defs.inc` below `DPC_STATUS_SET_XBUS` to use it:
`DPC_STATUS_SET_XBUS    equ 0x0002`